bigbluebutton-html5/imports/startup/server/redis2x.js

```diff
@@ -40,29 +40,25 @@ class RedisPubSub2x {
     return this.emitter.on(...args);
   }
 
-  publish(channel, eventName, meetingId, payload = {}, header = {}) {
-    const header2x = {
-      name: eventName,
-      meetingId,
-    };
-
-    const msgHeader = header === {} ? header2x : header;
-
+  publish(channel, eventName, meetingId, payload = {}, userId = {}) {
```

Comment:
Why the last parameter name is `userId`? It should be `header` since you are merging it in the header of the envelope.